src/components/pages/Dashboard/Explore.tsx

```diff
@@ -1,38 +1,58 @@
-import React from "react";
+import React, { useContext, useEffect } from "react";
 import { useMediaQuery } from "react-responsive";
-import { mobileDeviceOrTabletWidth } from "../../../constants";
+import { Dimmer, Loader } from "semantic-ui-react";
+import { isMaintenanceMode, mobileDeviceOrTabletWidth } from "../../../constants";
+import { AppContext } from "../../../context";
+import { PageStateEnum } from "../../../types";
 import Map from '../../map/Map';
 import MobileComponents from '../../mobile/ExploreMobile';
+import MaintenanceModal from "../../shared/MaintenanceModal";
 import LeftSidebar from "../../sidebar/left-sidebar/LeftSidebar";
 import RightSidebar from "../../sidebar/right-sidebar/RightSidebar";
-import MaintenanceModal from "../../shared/MaintenanceModal";
-import { isMaintenanceMode } from "../../../constants";
 
 export default function Explore() {
   const isMobileDeviceOrTablet = useMediaQuery({ maxDeviceWidth: mobileDeviceOrTabletWidth });
+  const [state, dispatch] = useContext(AppContext)
+
+  useEffect(() => {
+    dispatch({
+      type: "UPDATE_EXPLORE_IS_OPEN",
+      payload: true
+    })
+    return () => {
```

Comment:
what's the point of this here?

Reply:
it's a clean up function that runs when the component dismounts from the DOM